spec.html

```diff
@@ -36660,11 +36660,11 @@ <h1>
               1. Return _y_.
             1. Let _r_ be _m_(_x_, _d_).
             1. If _r_ is ~failure~, return ~failure~.
-            1. Let _y_ be _r_'s MatchState.
```

Comment:
Oh jeeze, I'd never noticed this before. I'm have no idea how this was supposed to make sense.